### PR TITLE
test: username change - 1

### PR DESCRIPTION
**Pull request by alisabzevari**
This is not a completed PR!

I want to check if I am going on the right way. If it is possible, please take a look at the test that I have written. I think it is ok except the part that I don't know how to simulate request of copying the whole user object to another one to couchdb. I think I have to add something after line 159 to simulate copying the whole user object.

### CODE REVIEW

File: tests/integration/routes/account/patch-account-test.js [hunk 1]
```diff
@@ -146,10 +146,65 @@ getServer(function (error, server) {
...
+            all_or_nothing: Joi.boolean().only(true).required(),
+            docs: Joi.array().items(
+              Joi.object({
+                'name': Joi.string().only('org.couchdb.user:newusername').required(),
+                'renamedTo': Joi.string().only('newusername').required(),
+                'username': Joi.any().forbidden()
+              }),
+              Joi.object({
+                '_id': Joi.string().only('org.couchdb.user:oldusername').required(),
+                '_deleted': Joi.boolean().only(true).required()
```
Comment (gr2m) [inline tests/integration/routes/account/patch-account-test.js]: 'renamedTo' shoudl be in the deleted `org.couchdb.user:oldusername` doc. Good idae with `'username': Joi.any().forbidden()`, can you add it to the 2nd document, too?

**Comment by gr2m** [comment]
Looking good so far @alisabzevari, just see my minor comment

**Comment by alisabzevari** [comment]
I've fixed your comments. But I think I didn't write test correctly because the [related route] (https://github.com/hoodiehq/hoodie-account-server/blob/issue/79/routes%2Faccount.js#L109) does not run correctly. `sessions.find` at [line 134](https://github.com/hoodiehq/hoodie-account-server/blob/issue/79/routes%2Faccount.js#L134) throws an exception and then it will throw `errors.INVALID_SESSION`

**Comment by gr2m** [comment]
closing in favor of https://github.com/hoodiehq/hoodie-account-server/pull/155

**Pull request closed without merging**